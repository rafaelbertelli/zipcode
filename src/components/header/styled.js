import styled from 'styled-components';

export const HeaderComponent = styled.div`
  background-color: #c7c7c7;
  color: #495057;
  margin-bottom: 15px;
  padding: 25px;
`;

export const Title = styled.h3`
  margin-bottom: 20px;
`;

export const Form = styled.form`
  align-items: center;
  display: flex;
  flex-wrap: wrap;
`;

export const Label = styled.label`
  margin-bottom: 10px;
`;

export const Input = styled.input`
  @media screen and (max-width: 425px) {
    width: 90px;
  }

  border: 1px solid #ced4da;
  border-radius: 3px;
  box-sizing: content-box;
  font-family: inherit;
  font-size: inherit;
  height: 35px;
  line-height: inherit;
  margin: 0 10px 10px;
  padding: 0px 10px;
  width: 200px;

  input[type='number']::-webkit-inner-spin-button,
  input[type='number']::-webkit-outer-spin-button {
    -webkit-appearance: none;
  }
`;

export const Button = styled.button`
  background-color: #007bff;
  border: 1px solid transparent;
  border-color: #007bff;
  border-radius: 3px;
  color: #fff;
  cursor: pointer;
  font-family: inherit;
  font-size: inherit;
  line-height: inherit;
  margin-bottom: 10px;
  padding: 8px 15px;
  transition: color 0.15s ease-in-out, background-color 0.15s ease-in-out,
    border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;

  &:hover:not(:disabled) {
    background-color: #0069d9;
    border-color: #0062cc;
    color: #fff;
  }

  &:disabled {
    background: #dddddd;
  }
`;
