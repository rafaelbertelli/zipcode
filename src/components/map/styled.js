import styled from 'styled-components';

export const MapComponent = styled.div`
  border: 2px solid #c7c7c7;
  padding: 20px;
  position: relative;
`;

export const ButtonClose = styled.button`
  cursor: pointer;
  background: none;
  border: none;
  height: 20px;
  opacity: 0.3;
  position: absolute;
  right: 10px;
  top: 10px;
  width: 20px;

  &:hover {
    opacity: 1;
  }

  &:before,
  &:after {
    background-color: #333;
    content: ' ';
    height: 20px;
    left: 10px;
    position: absolute;
    top: 0;
    width: 2px;
  }
  &:before {
    transform: rotate(45deg);
  }
  &:after {
    transform: rotate(-45deg);
  }
  &:active {
    opacity: 0.3;
  }
`;

export const SuperScription = styled.div`
  margin-bottom: 25px;

  p:first-of-type {
    line-height: 28px;
  }

  p {
    line-height: 18px;
  }
`;
