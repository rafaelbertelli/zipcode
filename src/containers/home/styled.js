import styled from 'styled-components';

export const HomeComponent = styled.div`
  color: #575756;
  margin: 0 auto;
  max-width: 768px;
  padding: 0 15px;
`;
