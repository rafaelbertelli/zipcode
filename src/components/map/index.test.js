import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { MapComponent } from './styled';

describe('Map Test Suite', () => {
  it('Should match snapshot', () => {
    const wrapper = shallow(<MapComponent />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
