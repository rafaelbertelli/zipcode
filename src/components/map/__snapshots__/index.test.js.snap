// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map Test Suite Should match snapshot 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  border: 2px solid #c7c7c7;
  padding: 20px;
  position: relative;
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
